tests/model_selection/test_hyperband.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import numpy as np
+import scipy.stats as stats
+import math
+
+from distributed import Client
+from distributed.utils_test import loop, cluster
+
+from dask_ml.datasets import make_classification
+from dask_ml.model_selection import HyperbandCV
+
+
+class ConstantFunction:
+    def _fn(self):
+        return self.value
+
+    def get_params(self, deep=None, **kwargs):
+        return {k: getattr(self, k) for k, v in kwargs.items()}
+
+    def set_params(self, **kwargs):
+        for k, v in kwargs.items():
+            setattr(self, k, v)
+        return self
+
+    def partial_fit(self, *args, **kwargs):
+        pass
+
+    def score(self, *args, **kwargs):
+        return self._fn()
+
+    def fit(self, *args):
+        pass
+
+
+@pytest.mark.parametrize("max_iter", [81])
+def test_info(max_iter, loop):
+    with cluster() as (s, [a, b]):
+        with Client(s['address'], loop=loop):
+            X, y = make_classification(n_samples=20, n_features=20, chunks=20)
+
+            model = ConstantFunction()
+
+            params = {'value': stats.uniform(0, 1)}
+            alg = HyperbandCV(model, params, max_iter=max_iter)
+            info = alg.info()
+            paper_alg_info = _hyperband_paper_alg(max_iter)
+
+            assert set(paper_alg_info.keys()) == set([b['bracket']
+                                                      for b in info['brackets']])
+            assert info['total_partial_fit_calls'] == sum(paper_alg_info.values())
+            for bracket in info['brackets']:
+                k = bracket['bracket']
+                assert bracket['partial_fit_calls'] == paper_alg_info[k]
+            assert info['total_partial_fit_calls'] == sum(b['partial_fit_calls']
+                                                          for b in info['brackets'])
+            assert info['total_models'] == sum(b['num_models']
+                                               for b in info['brackets'])
+
+
+def _hyperband_paper_alg(R, eta=3):
```

Comment:
Silly question: is it possible to replace this with hardcoded values?

Currently, the `test_info` test relies on the correctness of this implementation, which is itself untested ;)

Reply:
Good point. I have some hard coded values now. I have kept `_hyperband_paper_alg` so it's known that we have verified the implementation against the paper (and assert that the value it returns is equal to hard coded value).